__tests__/app.test.js

```diff
@@ -110,23 +110,73 @@ describe("/api/articles", () => {
       });
   });
 
-  test('should sort the articles by date in descending order', () => {
+  test("should sort the articles by date in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const articles = body.articles;
+        expect(articles).toBeSortedBy("created_at", {
+          descending: true,
+        });
+      });
+  });
+  test("GET: 404 - responds with an error message when given an invalid endpoint", () => {
+    return request(app)
+      .get("/api/articlesss")
+      .expect(404)
+      .then((body) => {
+        expect(body.res.statusMessage).toBe("Not Found");
+      });
+  });
+});
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET: 200 - should return the comments for the specified article id", () => {
     return request(app)
-    .get("/api/articles")
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({body}) => {
+        const comments = body.comments
+        expect(comments.length).toBe(11)
+        comments.forEach((comment) => {
+          expect(typeof comment.comment_id).toBe('number')
+          expect(typeof comment.votes).toBe('number')
+          expect(typeof comment.created_at).toBe('string')
+          expect(typeof comment.author).toBe('string')
+          expect(typeof comment.body).toBe('string')
+          expect(typeof comment.article_id).toBe('number')
+        })
```

Comment:
This is a great test!
On line `148` would you expect the `article_id` to be any number for this request or would it always be the number used in the path?